DailySelfie/app/src/main/java/com/teamagam/dailyselfie/MainActivity.java

```diff
@@ -135,4 +142,30 @@ private PictureInfo pictureInfoFromFile(File file) {
         return pictureInfo;
     }
 
+    private Notification buildDailyNotification() {
+        Notification.Builder notificationBuilder = new Notification.Builder(
+                getApplicationContext())
+                .setTicker(getString(R.string.notification_daily_text_ticker))
+                .setSmallIcon(R.drawable.ic_add_a_photo_black_24dp)
+                .setAutoCancel(true)
+                .setContentTitle(getString(R.string.notification_daily_title_content))
+                .setContentText(getString(R.string.notification_daily_text_content))
+                .setContentIntent(createDailySelfiePendingIntent());
+
+        return notificationBuilder.build();
+    }
+
+    private PendingIntent createDailySelfiePendingIntent() {
+        Intent dailySelfieIntent = new Intent(getApplicationContext(),
+                MainActivity.class);
+        dailySelfieIntent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
```

Comment:
what happens if the activity already exists? do you create new activity (maybe it's OK).
do you want to resume the current activity?